### PR TITLE
Create reportho blog post

### PR DESCRIPTION
**Pull request by itrujnara**
Blog post for reportho release

### CODE REVIEW

**Comment by itrujnara** [comment]
@netlify /blog/2024/reportho_intro

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Comment by itrujnara** [comment]
@mashehu Added the links, is it okay now?

**Comment by mashehu** [comment]
Almost, the link format in markdown is `[label](url)` not `(label)[url]`

File: sites/main-site/src/content/blog/2024/reportho_intro.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: 'A gentle introduction to nf-core/reportho'
+subtitle: 'The nf-core pipeline for comparing ortholog predictions'
+headerImage: '/assets/images/blog/reportho_intro/reportho_tube_map.png'
+headerImageAlt: 'Tube map of the reportho pipeline'
+pubDate: 2024-07-18T00:19:00+01:00
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/reportho_intro.mdx:6]:
```suggestion
pubDate: 2024-07-19T00:19:00+01:00
```

**Comment by itrujnara** [comment]
> Almost, the link format in markdown is `[label](url)` not `(label)[url]`

Oops, not enough hackathons. Fixed it now.

**Comment by itrujnara** [comment]
@nf-core-bot fix linting

**Comment by mashehu** [comment]
Good to go now. hit merge whenever you want  🙂

**Pull request merged**